app/controllers/user/booking_fields_controller.rb

```diff
@@ -1,8 +1,18 @@
 class User::BookingFieldsController < ApplicationController
   before_action :logged_in, :login_as_user, only: %i(new create)
-  before_action :get_booking_field, only: %i(pay demo_payment)
+  before_action :get_booking_field, only: %i(pay demo_payment update)
+  before_action :set_field_and_date, only: [:new]
```

Comment:
```suggestion
  before_action :set_field_and_date, only: :new
```
như vậy thôi không cần [] nữa ạ